### PR TITLE
Setup and instructions update

### PR DESCRIPTION
**Pull request by sameer-coder**
Changes include:

- Dockerfile added
- Readme instructions updated
- `1+N` renamed to `N+1`

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+version: '3.6'
+services:
+  postgres:
+    image: postgres:alpine
+    environment:
+      POSTGRES_PASSWORD: postgres
+    ports:
+      - '5433:5432'
```
Comment (Eomm) [inline docker-compose.yml:8]: Is `5433` intended or a typo?

Comment (simoneb) [reply docker-compose.yml:8]: good catch. this PR is missing this file https://github.com/nearform/the-fastify-workshop/blob/master/postgrator.json

Comment (sameer-coder) [reply docker-compose.yml]: Good catch @Eomm!

@simoneb postgrator.json is already in the repo

**Comment by simoneb** [comment]
There was a reason why it was that port. Can you guess why?

**Pull request merged**